docs/paper/dev/api/event-api/chat-event.md

```diff
@@ -0,0 +1,152 @@
+---
+slug: /dev/chat-events
+description: An outline on the AsyncChatEvent and how to handle it 
+---
+
+# Chat Events
+The Chat Event has evolved a few times over the years. 
+This guide will explain how to properly use the new `AsyncChatEvent` and its `ChatRenderer`.
+The `AsyncChatEvent` is an improved version of the old `AsyncPlayerChatEvent` that allows you to render chat messages individually for each player.
+
+## Understanding the Renderer
+
+Before we can start using the new chat event, we need to understand how the new renderer works.
+The renderer is Paper's way of allowing plugins to modify the chat message before it is sent to the player.
+This is done by using the `ChatRenderer` interface with its `render` method. Previously, this was done by using the `AsyncPlayerChatEvent` with its `setFormat` method.
```

Comment:
don't know why we'd bother mentioning the deprecated method here

Reply:
because it was requested:
https://github.com/PaperMC/docs/pull/264#discussion_r1382642954

Reply:
@kezz I think it's good to mention what it replaces. We frequently get questions about what the replacement for setFormat is on discord, enough that I think its warranted mentioning it somewhere, somehow.